structuretoolkit/visualize.py

```diff
@@ -807,3 +809,78 @@ def _get_flattened_orientation(
     flattened_orientation[:3, :3] = _get_orientation(view_plane)
 
     return (distance_from_camera * flattened_orientation).ravel().tolist()
+
+
+def plot_isosurface(
+    mesh,
+    value,
+    cell: Optional[Union[Atoms, list, np.ndarray, float]] = None,
+    structure_plot: Optional["plotly.graph_objs._figure.Figure"] = None,
+    isomin: Optional[float] = None,
+    isomax: Optional[float] = None,
+    surface_fill: Optional[float] = None,
+    opacity: Optional[float] = None,
+    surface_count: Optional[int] = 5,
+    colorbar_nticks: Optional[int] = None,
+    caps: Optional[dict] = dict(x_show=False, y_show=False, z_show=False),
+    colorscale: Optional[str] = None,
+    height: Optional[float] = 600,
+    camera: Optional[str] = "orthographic",
+    **kwargs,
+):
+    """
+    Make a mesh plot
+
+    Args:
+        mesh (numpy.ndarray): Mesh grid. Must have a shape of (3, nx, ny, nz).
+            It can be generated from structuretoolkit.create_mesh
+        value: (numpy.ndarray): Value to plot. Must have a shape of (nx, ny, nz)
+        cell (Atoms|ndarray|list|float|tuple): Cell, ignored if
+            `structure_plot` is given
+        structure_plot (plotly.graph_objs._figure.Figure): Plot of the
```

Comment:
I guess the type hint should show import directly from the public part of the API (`https://plotly.com/python-api-reference/generated/plotly.graph_objects.Figure.html`) and not `..._figure.Figure`